### PR TITLE
Add serialization tests

### PR DESCRIPTION
**Pull request by DavidEichmann**

### CODE REVIEW

File: byron/crypto/src/Cardano/Crypto/Signing/Redeem/Compact.hs [hunk 1]
```diff
@@ -70,7 +70,7 @@ instance ToCBOR CompactRedeemVerificationKey where
 
 instance FromCBOR CompactRedeemVerificationKey where
   fromCBOR = do
-    enforceSize "CompactRedeemVerificationKey" 1
+    enforceSize "CompactRedeemVerificationKey" 4
```
Comment (DavidEichmann) [inline byron/crypto/src/Cardano/Crypto/Signing/Redeem/Compact.hs:73]: Looks like there was a bug. This was a recent bug, so I don't think we need to worry too much about this bug surfacing in the wild.

Comment (JaredCorduan) [reply byron/crypto/src/Cardano/Crypto/Signing/Redeem/Compact.hs:73]: good catch!

**Review by JaredCorduan — APPROVED** [review]
Thanks @DavidEichmann !

**Comment by DavidEichmann** [comment]
Update: signed my commit with gpg

**Pull request merged**